source/Operation Guides/InterFlux Masternodes/Using Masternode Setup Tool/masternode-setup-tool.rst

```diff
@@ -0,0 +1,309 @@
+
+https://github.com/stratisproject/CirrusCore/releases
+
+3. Please ensure you take note of your Mnemonic and Passphrase. These
+   **will be required** later in the process
+
+3. Login to your STRAX Wallet
+
+.. image:: media/image2.jpeg
+   :width: 4.40157in
+   :height: 4.02362in
+
+4. Obtain a Receive Address and send 100,501 STRAX to this address
+
+.. image:: media/image3.png
+   :width: 4.40157in
+   :height: 4.02362in
+
+4. You will have to wait for 100% Wallet Synchronization before seeing
+   your balance
+
+5. Login to your Cirrus Core Wallet
+
+.. image:: media/image4.png
+   :width: 4.57874in
+   :height: 3.86614in
+
+6. Select Address 0
+
+.. image:: media/image5.png
+   :width: 4.58268in
+   :height: 3.86614in
+
+7. Obtain a Receive Address
+
+.. image:: media/image6.png
+   :width: 4.58268in
+   :height: 3.86614in
+
+8. Perform a Cross-Chain Transfer of **500.9 STRAX** to your Cirrus
+   Address
+
+.. image:: media/image7.png
+   :width: 4.40157in
+   :height: 4.02362in
+
+9. The Cirrus Core Wallet will receive 500.8~ CRS after 80 Blocks have
+   passed on the STRAX Blockchain (~1Hr)
+
+.. image:: media/image8.png
+   :width: 4.58268in
+   :height: 3.86614in
+
+10. Now, you must consolidate the STRAX Balance to ensure it all resides
+    in a single address. To do this, obtain another Receive Address and
+    send **100,000 STRAX**
+
+.. image:: media/image9.png
+   :width: 4.1811in
+   :height: 3.82283in
+
+5. You are sending back to an address that you own. Not an external
+   address.
+
+11. Your resulting balance and transaction history should look very
+    similar to the below
+
+.. image:: media/image10.png
+   :width: 4.1811in
+   :height: 3.82283in
+
+12. Wallet preparation is now complete; you should now be in position of
+    the following.
+
+1. **STRAX Wallet that contains 100,000+ STRAX Tokens.**
+
+2. **The Mnemonic and Passphrase for the aforementioned wallet.**
+
+3. **Cirrus Wallet that contains 500+ CRS Tokens.**
+
+4. **The Mnemonic and Passphrase for the aforementioned wallet.**
+
+13. You must now wait for **80 Confirmations** before the **CRS Tokens**
+    are received through the Cross-Chain Transaction, in addition, the
+    transaction that was made in **Step 10** also requires 500
+    Confirmations.
+
+InterFlux Masternode Registration 
+=======================================
+
+Whilst we wait for confirmations, the host that will run your STRAX
+Sidechain Masternode configured and prepared to launch the Masternode
+Setup Tool.
+
+Please ensure you meet the requirements set out at the `beginning of
+this guide <#sidechain-masternode-hardware-requirements>`__, before
+you continue.
+
+Obtain the Masternode Setup Tool installer
+---------------------------------------------------
+
+A tool has been developed to ease the setup process and ensure
+consistency, the tool can be obtained from the below URL.
+
+`Click here to download the Masternode Setup Tool <https://www.stratisplatform.com/>`_
```

Comment:
We can change it to https://github.com/stratisproject/MasternodeSetupTool/releases